src/components/form/widgets/input-field.ts

```diff
@@ -71,19 +72,19 @@ function getInputType(schema: any): InputType {
     return 'text';
 }
 
-function getStepSize(schema: any): 'any' | number {
+function getStepSize(schema: FormSchema): 'any' | number {
     if (isNumberType(schema) && schema.multipleOf) {
-        return parseFloat(schema.multipleOf) || 'any';
+        return parseFloat(String(schema.multipleOf)) || 'any';
```

Comment:
```suggestion
        return +schema.multipleOf || 'any';
```

Should this really return the _string_ `'any'`? 🤔

Reply:
Yes, the `step` property of `limel-input-field` is `"any" | number`